### ISSUE TITLE
Crash on resume android app with screen reader turned on

### ISSUE THREAD

**Issue by israelrios** [phonegap/phonegap-mobile-accessibility#40]
When the screen reader configuration is changed while the app is paused the app will crash on resume. The cause is a bug in the reflection code used to call "reload" on the Web View. This throws a IllegalArgumentException.

In MobileAccessibility.onResume(boolean) line 147-148, It should be:
`Object aView = getView.invoke(webView);
 Method reload = aView.getClass().getMethod("reload");
 reload.invoke(aView);`

**Comment by hirbod** [comment]
PR?
